shelldio.sh

```diff
@@ -18,6 +18,27 @@ my_stations="$HOME/.shelldio/my_stations.txt"
 
 ### Functions List
 
+validate_csv() {
+	awk 'BEGIN{FS=","}!n{n=NF}n!=NF{failed=1;exit}END{print !failed}' "$1"
+}
+
+validate_station_lists() {
+	if [[ $(validate_csv "$all_stations") -eq 0 ]]; then
```

Comment:
Πριν ελέγξεις την εγκυρότητα, έλεγξε την ύπαρξη.

```suggestion
if [ -f "$all_stations" ]; then
	if [[ $(validate_csv "$all_stations") -eq 0 ]]; then
```

Reply:
Makes sense